src/md4c.h

```diff
@@ -145,7 +145,9 @@ typedef enum MD_SPANTYPE {
 
     /* <u>...</u>
      * Note: Recognized only when MD_FLAG_UNDERLINE is enabled. */
-    MD_SPAN_U
+    MD_SPAN_U,
+
+    MD_SPAN_MENTION
```

Comment:
I would prefer renamimg it to `MD_SPAN_MENTIONLINK` for the sake of consistency e.g. with `MD_SPAN_WIKILINK` etc.